hack/update-clientset.sh

```diff
@@ -109,6 +113,12 @@ go run k8s.io/code-generator/cmd/register-gen \
   ${COMMON_FLAGS} \
   ${GATEWAY_INPUT_DIRS_SPACE}
 
+# This is an awful hack to make up for a deficiency in register-gen in k8s v1.32
```

Comment:
Is there an issue related to this problem that we can link here?

Reply:
This is a bit of a circular problem where I want to be able to link to this in the k/k issue as an illustration of the problem and solution. Will add a k/k issue link after that, but would like to merge this first.